### ISSUE TITLE
Character Not Logging In

### ISSUE THREAD

**Issue by nightf0x0012** [exodus4d/pathfinder#1009]
This started last night with my character not being followed on the map. I proceeded to logout and when I came back I get an error  saying "Access Denied Failed to load characterData from ESI". I went to Eveonline and under account settings and revoked tokens on all devices and getting the same error still. Not sure which side is having problems, eve or pathfinder but pathfinder is showing everything offline.  See screenshots attached.

![2020-12-18 06_27_36-Pathfinder](https://user-images.githubusercontent.com/62291139/102610025-7e4e6280-40fa-11eb-826c-9d7eb952d77f.png)
![2020-12-18 06_26_18-Pathfinder](https://user-images.githubusercontent.com/62291139/102610036-81e1e980-40fa-11eb-94d7-76e80b1e4abd.png)

**Comment by theshankyinstitute** [comment]
I am having the exact same problem as shown above. Some of my corp mates seem to be able to use it just fine, though they say it is not auto-updating for them when they jump wormholes. I noticed the problem yesterday, and it is still ongoing today (pacific time)

I setup PathFinder on my own web host and it seems to authenticate just fine and shows Tranq and ESI are online -- whereas the public pathfinder server does not.

**Comment by theshankyinstitute** [comment]
Further to previous, I can go to esi.evetech.net and sign in there no problems, so there's nothing wrong there.

**Comment by GQuantrill** [comment]
The error from the getServer status reads:
`You have been banned from using ESI. Please come chat with us in #esi on tweetfleet slack and we can work to a (truncated...`

CCP have obviously blocked the public pathfinder for whatever reason (most likely abuse)

**Comment by RSalo** [comment]
Have the same error. Should I do anything or this problem related to pathfinder?

**Comment by nightf0x0012** [comment]
It's related to pathfinder. we're all stuck atm

**Comment by GQuantrill** [comment]
You can run you're own local ones.  I use a docker container to run my own private pathfinder and it's working fine.

**Comment by theshankyinstitute** [comment]
It’s the public pathfinder that’s screwed. You can do something about it — setup your own instance! With all this going on I rented my own VPS and now our Corp has our very own instance of pathfinder working perfectly. I used LetsEncrypt for free SSL on it too, and registered a domain name for the Corp. Costs about $6 a month for the vps and $15 for the domain name. Corp members offered to send me ISK to help cover costs, and I directed them to send to Corp wallet so we can build more crap. Win/win!

**Comment by nightf0x0012** [comment]
Any workaround for us non-IT/CS knowledgeable people? I can just use tripwire until it's resolved if it comes to that.

**Comment by theshankyinstitute** [comment]
> Any workaround for us non-IT/CS knowledgeable people? I can just use tripwire until it's resolved if it comes to that.

Yeah either use other tech like Tripwire, or work out a deal with an IT/CS person to set up PF for you.

**Comment by dkovar** [comment]
Also setting up a private PF instance.

**Comment by nightf0x0012** [comment]
For those that aren't tech savvy enough to set this up yourself, check out https://forums.eveonline.com/t/it-authentication-platforms-pathfinder-hosting-and-more/173365 He set this up in less than 12 hours and it's payable in isk. Sorry for the mention but I would have been sitting here for days trying to figure this out on my own and I'm sure that there are others out there that could use the help.
